otx/algorithms/common/adapters/mmcv/nncf/utils.py

```diff
@@ -117,6 +117,30 @@ def model_eval(
         return eval_res[metric_name]
 
 
+def nncf_state_dict_pre_hook(state_dict, prefix, *args, **kwargs):
+    """NNCF-specific state dict pre-hook."""
```

Comment:
Could you please describe the reason this hook is needed? It may be helpful in the future.